vendor/knative.dev/pkg/version/version.go

```diff
@@ -33,7 +33,7 @@ const (
 	// NOTE: If you are changing this line, please also update the minimum kubernetes
 	// version listed here:
 	// https://github.com/knative/docs/blob/mkdocs/docs/snippets/prerequisites.md
-	defaultMinimumVersion = "v1.19.0"
+	defaultMinimumVersion = "v1.21.0"
```

Comment:
and this one as well https://github.com/openshift/knative-eventing/blob/release-v1.4/openshift/patches/005-k8s-min.patch